workspace/manifest-writer/src/index.ts

```diff
@@ -22,8 +23,20 @@ async function writeManifestFile (dir: string, manifest: Partial<WorkspaceManife
   })
 }
 
-export async function updateWorkspaceManifest (dir: string, updatedFields: Partial<WorkspaceManifest>): Promise<void> {
+export async function updateWorkspaceManifest (dir: string, updatedFields: Partial<WorkspaceManifest> & {
+  updatedCatalogs?: Catalogs
+  cleanupUnusedCatalogs?: boolean
+}): Promise<void> {
```

Comment:
it should be something like this:
```suggestion
export async function updateWorkspaceManifest (dir: string, opts: {
  updatedFields?: Partial<WorkspaceManifest>
  updatedCatalogs?: Catalogs
  cleanupUnusedCatalogs?: boolean
}): Promise<void> {
```